Society for Neuroscience Abstract 2016.js

```diff
@@ -0,0 +1,446 @@
+/** BEGIN TEST CASES **/
+var testCases = [
+	{
+		"type": "web",
+		"url": "http://www.abstractsonline.com/pp8/#!/4376/presentation/17079",
+		"items": [
+			{
+				"itemType": "conferencePaper",
+				"title": "Electrophysiology of rat motor thalamus reveals unique patterns of firing during movement",
+				"creators": [
+					{
+						"firstName": "M.",
+						"lastName": "Gaidica",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "C.",
+						"lastName": "Cyr",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "A.",
+						"lastName": "Hurst",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "A.",
+						"lastName": "Kamath",
+						"creatorType": "author"
+					},
+					{
+						"firstName": "D. K.",
+						"lastName": "Leventhal",
+						"creatorType": "author"
+					}
+				],
+				"date": "November 15, 2017",
+				"abstractNote": "The thalamus is commonly regarded as a relay nucleus, but it is increasingly appreciated that it is a critical node in shaping normal and pathological motor behaviors. It remains unclear how its primary inputs from the basal ganglia, cerebellum, and cortex differentially contribute to the cortical-bound output of the “motor thalamus”. To determine behavioral correlates of neuronal activity in the rat motor thalamus we performed electrophysiology during a cued choice reaction time task. We found task-specific firing rate modulations for all behavioral events based on single unit neuronal classifications. This included neurons that rapidly increased their firing rate in response to sensory cues, neurons that showed tonically sustained firing during all movements, and neurons that showed rhythmic increases in firing rate precisely at movement onset. We performed a detailed investigation of neuronal activity specifically associated with the movement onset population as it has specific implications to movement disorders characterized by the inability to initiate movements (e.g. Parkinson’s disease). These neurons’ firing rates are depressed during a hold period prior to cued movement initiation. This is consistent with standard thalamocortical firing rate models which suggest GABAergic afferents from the basal ganglia regulate movement through disinhibition. Secondly, the burst frequency across these neurons was highly regular and consistently in the theta band (4-8 Hz). Finally, many of these neurons fired more strongly for contralateral movements. Ongoing work is directed at determining whether these functional classifications are correlated with distinct anatomic or physiologic features (e.g., cerebellar vs basal ganglia afferents), and how these activity patterns are generated.",
+				"conferenceName": "Society for Neuroscience",
+				"extra": "Session: Session 689 - Subcortical Physiology and Regulation of Behavior; Session Type: Poster",
+				"language": "eng",
+				"libraryCatalog": "Society for Neuroscience Abstract 2016",
+				"pages": "689.12/DP07/II19",
```

Comment:
It's not pages per se. 

But when you cite an abstract in paper, holding presentation id in `pages` is a good compromise. Actually, it works really well if we choose `journalArticile`. See examples below.

It's the question of do we prefer to have abstracts data which are not citable but categorized properly or those which are citable but categorized in an awkward way. 

Now I'm personally inclined to go with `journalArticle`. Because if they cannot be cited, they are a bit pointless. Tell me what you think.


## output from `conferencePaper` in The Journal of Neuroscience style: 

Pai L, Vogt D, Perez AC, Wimer M, McKinsey G, Hu JS, Sandberg M, Pla R, Goodrich LV, Paz JT, Rubenstein JL (2017) Mafb and c-Maf control the balanced neurogenesis of MGE-derived PV+ and SST+ GABAergic cortical interneurons. In: Society for Neuroscience Abstract, pp 282.18/B8 Available at: http://www.abstractsonline.com/pp8/#!/4376/presentation/30154 [Accessed September 28, 2017].

This is awkward, and unacceptable.

## output from `journalArticle` in The Journal of Neuroscience style: 

Pai L, Vogt D, Perez AC, Wimer M, McKinsey G, Hu JS, Sandberg M, Pla R, Goodrich LV, Paz JT, Rubenstein JL (2017) Mafb and c-Maf control the balanced neurogenesis of MGE-derived PV+ and SST+ GABAergic cortical interneurons. Soc Neurosci Abstr:282.18/B8.

This is pretty much how it should appear in Reference section of a paper

Reply:
I've never done Pull Request for a group project. I'm quite impressed by the quality of your thorough and detailed code review with deep understanding. It's a great learning experience for me. Hopefully, I can commit an update soon.

Reply:
Okay, we should discuss the pages and itemType with @adam3smith further.

Reply:
Take the time you need to update the points. And let us know by write a short comment.

Reply:
Thanks for consideration. I've done the changes apart from itemType.

Reply:
Okay, here in GitHub I still see only your initial commit. Make sure you did `git pull` to your GitHub fork.

Reply:
I did `push` to my fork. Would that be enough?

Reply:
Yes, it is. 🙇